lib/Partitioner/Partitioner.cpp

```diff
@@ -52,6 +50,8 @@ static llvm::cl::opt<bool>
                   llvm::cl::desc("Enable dumping the graph of each partitions"),
                   llvm::cl::init(false), llvm::cl::cat(PartitionerCat));
 
+#include <fstream>
```

Comment:
It doesn't look like `fstream` is used.  (If it is, put it with the other `#include`s above.